src/main/java/frc/robot/subsystems/motorsubsystem/MotorCommandBuilder.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.motorsubsystem;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+import java.util.function.Supplier;
+
+public class MotorCommandBuilder {
+
+	MotorSubsystem motorSubsystemPosition;
+
+	public MotorCommandBuilder() {
+		motorSubsystemPosition = new MotorSubsystem(motorSubsystemPosition.motor, motorSubsystemPosition.getLogPath());
+	}
```

Comment:
wont work (the motor subsystem you have is null rn) and also DI, get a MotorSubsystem in the constructor